### PR TITLE
Refactor/dockers/apply build for 2 platform

### PR DESCRIPTION
**Pull request by vankichi**
### Description:

SSIA

### Versions:

- Go Version: 1.21.5
- Docker Version: 20.10.8
- Kubernetes Version: v1.28.4
- NGT Version: 2.1.5

### Checklist:

- [x] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/main/CONTRIBUTING.md) document and completed [our CLA agreement](https://cla-assistant.io/vdaas/vald).
- [x] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?

### CODE REVIEW

File: Makefile.d/build.mk [hunk 1]
```diff
@@ -329,30 +329,26 @@ cmd/tools/benchmark/job/job: \
...
-		-X '$(GOPKG)/internal/info.CGOEnabled=$${CGO_ENABLED}' \
+		-X '$(GOPKG)/internal/info.CGOEnabled=${CGO_ENABLED}' \
 		-X '$(GOPKG)/internal/info.NGTVersion=$(NGT_VERSION)' \
 		-X '$(GOPKG)/internal/info.BuildCPUInfoFlags=$(CPU_INFO_FLAGS)' \
 		-buildid=" \
 		-mod=readonly \
 		-modcacherw \
 		-a \
-		-tags "cgo osusergo netgo" \
+		-tags "osusergo netgo static_build" \
```
Comment (kpango) [inline Makefile.d/build.mk:351]:
```suggestion
		-tags "cgo osusergo netgo static_build" \
```

File: Makefile.d/build.mk [hunk 2]
```diff
@@ -372,15 +368,15 @@ cmd/tools/benchmark/operator/operator: \
...
+		--ldflags "-w -extldflags=-static \
+		-X '$(GOPKG)/internal/info.CGOEnabled=${CGO_ENABLED}' \
 		-X '$(GOPKG)/internal/info.NGTVersion=$(NGT_VERSION)' \
 		-X '$(GOPKG)/internal/info.BuildCPUInfoFlags=$(CPU_INFO_FLAGS)' \
 		-buildid=" \
 		-mod=readonly \
 		-modcacherw \
 		-a \
-		-tags "cgo osusergo netgo" \
+		-tags "osusergo netgo static_build" \
```
Comment (kpango) [inline Makefile.d/build.mk:379]:
```suggestion
		-tags "cgo osusergo netgo static_build" \
```

**Review by kpango — CHANGES_REQUESTED** [review]

File: go.sum [hunk 3]
```diff
@@ -219,6 +219,8 @@ github.com/aws/aws-sdk-go-v2/service/sts v1.26.3 h1:KfREzajmHCSYjCaMRtdLr9boUMA7
 github.com/aws/aws-sdk-go-v2/service/sts v1.26.3/go.mod h1:7Ld9eTqocTvJqqJ5K/orbSDwmGcpRdlDiLjz2DO+SL8=
 github.com/aws/smithy-go v1.18.1 h1:pOdBTUfXNazOlxLrgeYalVnuTpKreACHtc62xLwIB3c=
 github.com/aws/smithy-go v1.18.1/go.mod h1:NukqUGpCZIILqqiV0NIjeFh24kd/FAa4beRb6nbIUPE=
+github.com/benbjohnson/clock v1.3.5 h1:VvXlSJBzZpA/zum6Sj74hxwYI2DIxRWuNIoXAzHZz5o=
```
Comment (kpango) [inline go.sum:222]: can you describe about this libs?
go mod why github.com/benbjohnson/clock

Comment (vankichi) [reply go.sum:222]: @kpango 
It is because `go.uber.org/ratelimit` requires `benbjonshon/clock`, and I think I missed `go.mod` file before rebasing.

Comment (kpango) [reply go.sum:222]: ok

**Review by kpango — APPROVED** [review]

**Pull request merged**